src/App.tsx

```diff
@@ -18,6 +18,8 @@ function App() {
   const setCurrentView = useViewStore((state) => state.setCurrentView);
   const hasMultiColumnMeasures = useDataStore((state) => state.hasMultiColumnMeasures());
 
+  const disableUploadNextButton = useDataStore((state) => state.uploadedDataTableFileName == null);
```

Comment:
is the `==` intentional here? i.e. can `uploadedDataTableFileName` be undefined?

Reply:
Yes, it's intentional. No, the default value is set to `null` by us in the store.